src/CoreWCF.Http/tests/SimpleWSHTTPTest.cs

```diff
@@ -286,12 +277,8 @@ public void WSHttpRequestImpersonateFailsWithoutAuthentication()
             }
         }
 
-        [Fact]
+        [WindowsNetCoreOnlyFact]
         [Description("no-security-with-an-anonymous-client-using-impersonation-httpsys")]
```

Comment:
WindowsOnly, should work on .NET Framework

Reply:
done